libcaf_io/test/remote_actor_udp.cpp

```diff
@@ -207,7 +207,12 @@ CAF_TEST(multiple_endpoints_udp) {
       }
     };
   });
-  server_sys.middleman().publish_udp(mirror, 12345);
+  // Publish server actor.
+  auto eport = server_sys.middleman().publish_udp(mirror, 0);
```

Comment:
Simpler:

`auto port = unbox(server_sys.middleman().publish_udp(mirror, 0));`

Reply:
It seems to be local to the [config_option test](https://github.com/actor-framework/actor-framework/blob/master/libcaf_core/test/config_option.cpp#L91).

Reply:
Nope, just include `dsl.hpp`: https://github.com/actor-framework/actor-framework/blob/master/libcaf_test/caf/test/dsl.hpp#L627

Reply:
Sorry, my bad. You're not using `test_coordinator_fixture`.